database-commons/pom.xml

```diff
@@ -59,6 +59,12 @@
       <artifactId>cdap-api</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.mockrunner</groupId>
+      <artifactId>mockrunner-jdbc</artifactId>
```

Comment:
where is this being used?

Reply:
I’m using mockrunner-jdbc to mock ResultSet and ResultSetMetaData. They are used in CommonFieldsValidatorTest and AbstractDBSinkTest classes.